server/routers/favoritesRouter.js

```diff
@@ -39,11 +39,10 @@ router.get('/:id', async (req, res) => {
 
 
 /************* Delete Favorite *************/
-router.delete('/:id', (req, res) => {
-    const { id } = req.params
-
-    if (id) {
-        favorites.remove(id)
+router.delete('/:symbol', (req, res) => {
```

Comment:
symbols are going to be repeated in the table so i'm not sure about using it to delete an entry

Reply:
i meant to type this in the helper file. sorry